src/info.rs

```diff
@@ -428,7 +428,7 @@ impl Info {
 
             let branches = String::from_utf8_lossy(&output.stdout);
 
-            branches.lines().count()
+            branches.lines().count() - 1 //Exclude origin/HEAD -> origin/master
```

Comment:
If onefetch is run in a repository with no branches, such as a repo that was just created but with no commits, this may result in an unsigned integer underflow error